### ISSUE TITLE
Guard against empty tags

### ISSUE THREAD

**Issue by nelsonkopliku** [trento-project/web#305]
Currently our tagging system allows empty tags.
If I press enter on a tag input without typing any text, trento allows that tag to be created.

I would argue about the usefulness of an empty tag already here.

Then this opens the issue of not being able to delete those tags.
When I refresh the page I get the items with the empty tag pill, and when I try to delete it, the backend goes 404 because of a non-existent deletion endpoint.

Simply put

the deletion endpoint should be something like
```
DELETE /api/clusters/6bd7ec60-8cb1-5c6b-a892-29e1fd2f8380/tags/<the-tag-value>
```

When `<the-tag-value>` is empty, the computed enpoint being called is 
```
DELETE /api/clusters/6bd7ec60-8cb1-5c6b-a892-29e1fd2f8380/tags/
```

which does not exist.

![tags-bug](https://user-images.githubusercontent.com/8167114/162021358-13260888-4a71-47d7-88d6-7845e3e9a203.gif)

**Comment by nelsonkopliku** [comment]
The backend part has been addressed here https://github.com/trento-project/web/pull/364

**Comment by arbulu89** [comment]
Fix for frontend: https://github.com/trento-project/web/pull/390
